docs/src/tut/mkt/index.md

```diff
@@ -0,0 +1,1323 @@
+:::
+
+`{!cmd} ./reach run` produces an array by default. It looks like this:
+
+```reach
+[ '/usr/local/bin/node', '/app/index.mjs' ]
+```
+
+So any string added to the `{!cmd} ./reach run index` command, will be added to that array.
+For example, `{!cmd} ./reach run index seller` will produce:
+
+```reach
+[ '/usr/local/bin/node', '/app/index.mjs', 'seller' ]
+```
+
+The item(s) that come after the second item in the array, tells us the variable that has been passed in.
+
+To see this working, type the following on line 4 of the `index.mjs` file:
+
+```reach
+console.log(process.argv);
+```
+
+Run `{!cmd} ./reach run index seller` in the terminal.
+
+The output it produces will look like this:
+
+```reach
+[ '/usr/local/bin/node', '/app/index.mjs', 'seller' ]
+Welcome to the Market
+Contract info: {"type":"BigNumber","hex":"0xba"}
+List of products for sale:
+1. Potatoes at 10 per bag.
+2. Carrots at 10 per bunch.
+3. Corn at 5 per ear.
+Buyer wants Potatoes
+Buyer agrees to buy 95 bags of Potatoes
+Seller agrees to sell 95 bags of Potatoes
+```
+
+The first line of the output is the line of interest. 
+That will be leveraged in giving the `{!rsh} Participant`s autonomy as the project for this section is developed.
+
+That is the basics of how the `{!cmd} ./reach run` command works. 
+
+
+### {#tut-mkt-interaction-why} Why Build An Interactive DApp
+The Basic DApp has been working fine and the business has grown but the `seller` no longer enjoys a monopoly - he now has competitors. 
+The `buyer`s have complained of either not getting what they want or not getting the quantity they want.
+
+He realizes that for him to compete, he has to make transactions more flexible for the `buyer`s. 
+To do this, the application must be improved.
+
+You're being commissioned once again to improve the DApp. Following the steps below, the application was completed:
+
+### {#tut-mkt-Interaction-basic} Basic Interactive DApp
+To make it easy to follow, we'll begin with new files. 
+
+Type the following code in the backend file:
+
+```reach
+load: /examples/mkt-4-interaction-basic/index.rsh
+range: 1 - 34
+```
+
+This code looks familiar. A few changes were made:
+
+* `choice` and `quantity` has been removed.
+
+* `commonInteract` is currently empty. 
+It has also been destructured into the `sellerInteract` and `buyerInteract` `{!rsh} Object`.
+This will produce the same result as before when it was destructured inside of the `{!rsh} Participant` definition.
+
+* `shop` `{!rsh} function` now returns an `{!rsh} Object` containing `prodNum` and `prodAmt` instead of `choice` and `quantity`.
+
+* Since `commonInteract` has already been destructured into the `sellerInteract` and `buyerInteract` `{!rsh} Object`, 
+`sellerInteract` and `buyerInteract` was passed to the `{!rsh} Participant` definition without destructuring.
+
+* Finally, `Seller` and `buyer` has been changed to `S` and `B`.
+
+That's all for the backend for now. Go to the frontend.
```

Comment:
Instead of an order, which can be off-putting, we might use "Now, let's move to the frontend" or something similar.